web/tendrl/user_password_edit_dialog.rst

```diff
@@ -0,0 +1,69 @@
+Change user password at user "Edit" page
+*****************************************
+
+:authors: 
+          - mkudlej@redhat.com
+
+Description
+===========
+
+Change user password at user "Edit" page for these user types:
+ 1. *Admin* user
+ 2. *Power User* user
+ 3. *Read-only* user
+
+Design describing "Edit" page https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805
```

Comment:
Travis is complaining that the anchor `/screens/226063805` is not found but this probably is not a issue on our site. https://travis-ci.org/usmqe/usmqe-testdoc/builds/274153140?utm_source=github_status&utm_medium=notification

Reply:
I just disabled linkcheck for anchors from invisionapp.com here: https://github.com/usmqe/usmqe-testdoc/commit/d6d0e50ef6a339296aabfbb840bd7808b4a7321a